content/en/synthetics/guide/api_test_timing_variations.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Understanding API test timings and troubleshooting their variations
+kind: documentation
+description: Understanding API test timings and troubleshooting their variations
+aliases:
+- /synthetics/api_test_timing_variations
+further_reading:
```

Comment:
Let's maybe add one or two other links (like the two first links in https://github.com/DataDog/documentation/pull/11017/files for isntance)